helpers/logger.py

```diff
@@ -1,9 +1,13 @@
 import logging
 
+
 def setup_logger(logger_name, log_level="warning"):
     # type: (str, str) -> logging.Logger
-    # todo : read level from config here
-    logger_level_attr = getattr(logging, log_level.upper())
+    try:
```

Comment:
I suggest having a whitelist of arguments (checked after uppercasing) - there are more attributes in logging than the 5 we accep, so AttributeError wont' always trigger on wrong values. Also, I highly suggest printing a warning (or even an error) when an app doesn't init the logging in the right way - this is not something that should pass silently!